modules/diskq/diskq-grammar.ym

```diff
@@ -64,13 +64,21 @@ DiskQueueOptions *last_options;
 %token KW_QOUT_SIZE
 %token KW_DIR
 
+%type <num> nonnegative_integer64
```

Comment:
This is no place for a general type like this. Move it please to the `lib/cfg-grammar.y`.

Reply:
The reason I put it here is that diskq is the *only* user, and it made sense to have it where it is used. I can move it, of course, just felt better to not pollute `cfg-grammar.y` with it. Doubly so, because then one could rightly campaign for a `positive_integer64` too, because that's nearby and similar - yet, would be completely unused.

Reply:
The usage and definition make sense. But the thin layer of abstraction for the basic types are stronger binding force to me. Of course the `source`/`destination` are there to ruin the picture.

Exactly right about `positive_integer64` :)
Although the dead code created by it is less desirable to me than the symmetry. (Not strictly against to have it.)